### DISCUSSION TITLE
Add possibility to disable periodically the new tab feature

### DISCUSSION THREAD

**Discussion by AlexDalencourt** [dailydotdev/daily#1248] [Feature Requests]
Hello,

I use the daily.dev app on my smartphone every days but I like to use it on my browser too. 

The problem is that I need the default google new tab for access to my company tools. The pause feature is usefull but he don't allow to make pause periodically, I need every times to make the pause manually.

I have some ideas like :
* Make a possibility to show the new tab feature one time by hour (could be parameterized)
* Split the screen with new tab feature and the default browser new tab to allow acces to default company apps
* Enable to desactivate totaly the new tab feature

Without one of this 3 features I think I will not use the browser extension because it will be painfull to change every times my configuration. I think I am not alone in that case.

Thank you for you consideration

PS : I use chromium as browser

**Comment by idoshamun** [comment]
Hi @AlexDalencourt,

Thanks for the idea! Have you considered using our webapp? You can install it as a PWA on desktop even
